src/Symfony/Component/Messenger/Bridge/Amqp/Transport/Connection.php

```diff
@@ -214,6 +217,10 @@ public static function fromDsn(string $dsn, array $options = [], AmqpFactory $am
             return $queueOptions;
         }, $queuesOptions);
 
+        if ($useAmqps && !self::hasCaCertConfigured($amqpOptions)) {
+            throw new SSLConnectionException("No ca certificate has been provided, set it in php files configuration 'amqp.cacert' or through DSN with cacert parameter to use SSL.");
```

Comment:
This should use `InvalidArgumentException` like other validation errors during the DSN parsing.

Reply:
That was one of my question. Thank you !